packages/grid/src/elements/pane/Pane.tsx

```diff
@@ -16,17 +18,33 @@ const PaneComponent = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>
   ({ children, ...props }, ref) => {
     const [paneId, setPaneId] = useState<string>();
 
+    const observerRef = useRef<HTMLDivElement>(null);
+    const { width = 1, height = 1 } = useResizeObserver<HTMLDivElement>({ ref: observerRef });
+
+    const isVisible = useMemo(() => {
+      if (observerRef.current) {
+        if (width <= 1) {
+          return false;
+        } else if (height <= 1) {
+          return false;
```

Comment:
Might be splitting hairs (pixels), but why aren't we comparing with `0`?

Reply:
Fair point, this was an err on my end from the default value of `1` set on the `width` and `height` properties provided by the `useResizeObserverHook`. I've updated to use `0` instead.